contracts/market-wrapper/NounsBuilderMarkerWrapper.sol

```diff
@@ -0,0 +1,132 @@
+    }
+
+    // ======== External Functions =========
+
+    /**
+     * @notice Determine whether the current auction is valid
+     * Parameters auctionId and nftContract are unused because there's at most one ongoing auction for Nouns Builder DAOs
+     * @return TRUE if the tokenId matches the current token for sale, and the auction is neither settled nor paused
+     */
+    function auctionIdMatchesToken(
+        uint256, // auctionId
+        address, // nftContract
+        uint256 tokenId
+    ) public view override returns (bool) {
+        AuctionState memory state = _getAuctionState();
+        return (
+            state.tokenId == tokenId
+            && !state.settled
+            && !market.paused() 
+        );
+    }
+
+    /**
+     * @notice Calculate the minimum next bid for this auction
+     * Parameter auctionId is unused because there's at most one ongoing auction for Nouns Builder DAOs
+     * @return minimum bid amount
+     */
+    function getMinimumBid(uint256) external view override returns (uint256) {
+        AuctionState memory state = _getAuctionState();
+        if (state.highestBidder == address(0)) {
+            // if there are NO bids, the minimum bid is the reserve price
+            return reservePrice;
+        } else {
+            // if there ARE bids, the minimum bid is the current bid plus the increment buffer
+            return state.highestBid + (state.highestBid * minBidIncrement / 100);
+        }
+    }
+
+    /**
+     * @notice Query the current highest bidder for this auction
+     * Parameter auctionId is unused because there's at most one ongoing auction for Nouns Builder DAOs
+     * @return highest bidder
+     */
+    function getCurrentHighestBidder(uint256) external view override returns (address) {
+        AuctionState memory state = _getAuctionState();
+        return state.highestBidder;
+    }
+
+    /**
+     * @notice Submit bid to Market contract
+     * Parameter auctionId is unused because there's at most one ongoing auction for Nouns Builder DAOs
+     */
+    function bid(uint256, uint256 bidAmount) external override {
+        AuctionState memory state = _getAuctionState();
+        market.createBid{ value: bidAmount }(state.tokenId);
+    }
+
+    /**
+     * @notice Determine whether the auction has been finalized
+     * Parameter auctionId is unused because there's at most one ongoing auction for Nouns Builder DAOs
+     * @return TRUE if the auction has been finalized
+     */
+    function isFinalized(uint256) external view override returns (bool) {
+        AuctionState memory state = _getAuctionState();
+        return state.settled;
+    }
+
+    /**
+     * @notice Finalize the results of the auction
+     * Parameter auctionId is unused because there's at most one ongoing auction for Nouns Builder DAOs
+     */
+    function finalize(uint256) external override {
+        market.settleAuction();
+    }
```

Comment:
How will this function be used? Should I leave the body empty? None of the PartyBid members will have permissions to settle the current auction.

Reply:
It seems like this function is only used in tests?